src/stopping_solve.jl

```diff
@@ -0,0 +1,97 @@
+struct NNStopping{C,O,S,E} <: NeuralNetDiffEqAlgorithm
+    chain::C
+    opt::O
+    sdealg::S
+    ensemblealg::E
+end
+NNStopping(chain  ; opt=Flux.ADAM(0.1) , sdealg = EM() , ensemblealg = EnsembleThreads()) = NNStopping(chain , opt , sdealg , ensemblealg)
+
+function DiffEqBase.solve(
+    prob::OptimalStoppingProblem,
+    alg::NeuralNetDiffEqAlgorithm;
+    abstol = 1f-6,
+    verbose = false,
+    maxiters = 300,
+    trajectories = 1000,
+    save_everystep = false,
+    dt,
+    kwargs...
+    )
+
+    tspan = prob.tspan
+    sigma = prob.sigma
+    μ = prob.mu
+    g = prob.g
+    u0 = prob.u0
+    ts = tspan[1]:dt:tspan[2]
+    N = size(ts)[1]
+    T = tspan[2]
+
+    m  = alg.chain
+    opt    = alg.opt
+    sdealg = alg.sdealg
+    ensemblealg = alg.ensemblealg
+    
+    prob = SDEProblem(μ,sigma,u0,tspan)
+    ensembleprob = EnsembleProblem(prob)
+    sim = solve(ensembleprob, sdealg, ensemblealg, dt=dt,trajectories=trajectories,adaptive=false)
+    payoff = []
+    times = []
+    iter = 0
+    sump = 0
+    for u in sim.u
+        X = u.u
+        a = []
+        function Un(n , X )
+            if size(a)[1] >= n
```

Comment:
where is `n` defined?

Reply:
n is the parameter. its passed here https://github.com/ashutosh-b-b/NeuralNetDiffEq.jl/blob/305a8c0c8a7a867f7c25bb13390ee88c0475d1f6/src/stopping_solve.jl#L62